lib/linguist/generated.rb

```diff
@@ -525,5 +526,19 @@ def generated_grpc_cpp?
       return false unless lines.count > 1
       return lines[0].start_with?("// Generated by the gRPC")
     end
+    
+    # Internal: Is this an owl-generated Dart file?
+    #
+    # An own generated file contains:
+    # // GENERATED CODE - DO NOT MODIFY
+    # on the first line.
+    #
+    # Return true or false
+    def generated_dart?
+      return false unless extname == '.dart'
+      return false unless lines.count > 1
+      return true if lines[0].include?("Generated code. Do not modify.")
+      return lines[1].include?("GENERATED CODE - DO NOT MODIFY")
```

Comment:
I think these two lines can probably be merged into one case-insensitively checking for “generated code”.

Reply:
Back to my previous comment... the last two lines can be done in a single check using:

```
lines.first(2).any? { |l| l.downcase =~ /generated code.+do not modify/  }
```

... however it does have a few assumptions and does raise a few questions...

1. Given the comment you've added for this section, do we even need the check for `"Generated code. Do not modify."`?
2. Your comment says the `// GENERATED CODE...` line will be on the _first_ line, yet you're testing for it on the second line. Which is it or is it likely to be in either the first or second line?
3. Will the "generated code" line _always_ be in the first two lines or could it be a little further down too, for example on line 5?